recipes/spikingjelly/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "spikingjelly" %}
+{% set version = "0.0.0.0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spikingjelly-{{ version }}.tar.gz
+  sha256: 5d04a84e044e994e5b73c0f7f5b96756d3d5f935d088a9c7e04a4c58c44eb65c
+  patches:
+    - fix_no_requirements.patch
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - matplotlib-base
+    - numpy
+    - onnx
+    - python
+    - scipy
+    - pytorch >=1.5.0
+    - torchvision
+    - tqdm
+    - cupy  # [linux]
```

Comment:
This package can probably be noarch, no? If the only reason not to is the lack of cupy on Windows we can let the Win CI fail here and merge. The conda solver will tell users that they cannot install it on Windows and, if we ever get cupy on Windows, this would package will be ready to be installed.

Reply:
Sure, just changed that :)